core/templates/staff/dashboard.html

```diff
@@ -13,9 +13,6 @@ <h1>Global Statistics</h1>
         <li><span class="dash-label">Registered users:</span> <span class="dash-value">{{num_users|intcomma}}</span></li>
       </ul>
     </div>
-    <div class="global-section-footer">
-      <a href="{% url 'internal-leaderboard' %}" class="button is-medium" target="_blank">View Leaderboard</a>
```

Comment:
@danlamanna Do we want any other button here instead? Is there a functionality which is truly global for all challenges?

Reply:
Nope.